kaldb/src/main/java/com/slack/kaldb/chunkManager/IndexingChunkManager.java

```diff
@@ -277,6 +279,9 @@ private ReadWriteChunk<T> getOrCreateActiveChunk(
               LuceneIndexStoreImpl.makeLogStore(
                   dataDirectory, indexerConfig.getLuceneConfig(), meterRegistry);
 
+      // this returns the latest active chunk on which we will calculate disk size
+      activeChunkDirectory = logStore.getDirectory().toFile();
```

Comment:
Also, let's move this logic into the chunk roll over strategy since we don't need that info for indexing logic.